spec/migrations/20170710132935_convert_custom_button_to_classes_spec.rb

```diff
@@ -0,0 +1,88 @@
+require_migration
+
+describe ConvertCustomButtonToClasses do
+  context 'CustomButton' do
+    let(:params) { custom_button.options }
+    let!(:custom_button) { migration_stub(:CustomButton).create!(create_params) }
+
+    migration_context :up do
+      let(:create_params) { {:options => {:button_image => 1} } }
+
+      it 'sets the image class and color' do
+        migrate
+        custom_button.reload
+
+        expect(params).to have_key(:button_icon)
+        expect(params).to have_key(:button_color)
+        expect(params).not_to have_key(:button_image)
+      end
+
+      context 'button with default color' do
```

Comment:
`context` is good for grouping multiple specs. So here is only 1 spec, may get flagged by other reviewers (I am feeling like a :trollface: to ask more ...)
you can move the context text into the `it` statement.